src-tauri/Cargo.toml

```diff
@@ -8,35 +8,38 @@ repository = "https://github.com/subspace/subspace-desktop"
-sp-core = { version = "6.0.0", git = "https://github.com/subspace/substrate", rev = "b49d8ee54f4bd6e4b5b48f45d08f75705fad95b4" }
-sp-panic-handler = { version = "4.0.0", git = "https://github.com/subspace/substrate", rev = "b49d8ee54f4bd6e4b5b48f45d08f75705fad95b4" }
-subspace-core-primitives = { git = "https://github.com/subspace/subspace", rev = "61c60e35fc03e286bb91fd61f799c0e279dcb351" }
-subspace-farmer = { git = "https://github.com/subspace/subspace", rev = "61c60e35fc03e286bb91fd61f799c0e279dcb351" }
-subspace-fraud-proof = { git = "https://github.com/subspace/subspace", rev = "61c60e35fc03e286bb91fd61f799c0e279dcb351" }
-subspace-runtime = { git = "https://github.com/subspace/subspace", rev = "61c60e35fc03e286bb91fd61f799c0e279dcb351" }
-subspace-service = { git = "https://github.com/subspace/subspace", rev = "61c60e35fc03e286bb91fd61f799c0e279dcb351" }
-subspace-solving = { git = "https://github.com/subspace/subspace", rev = "61c60e35fc03e286bb91fd61f799c0e279dcb351" }
+sc-chain-spec = { version = "4.0.0-dev", git = "https://github.com/subspace/substrate", rev = "1399afdcd8ab4d7fad149c96f9cadcaf04b94f86" }
+sc-client-api = { version = "4.0.0-dev", git = "https://github.com/subspace/substrate", rev = "1399afdcd8ab4d7fad149c96f9cadcaf04b94f86" }
+sc-consensus-slots = { version = "0.10.0-dev", git = "https://github.com/subspace/substrate", rev = "1399afdcd8ab4d7fad149c96f9cadcaf04b94f86" }
+sc-executor = { version = "0.10.0-dev", git = "https://github.com/subspace/substrate", rev = "1399afdcd8ab4d7fad149c96f9cadcaf04b94f86", features = ["wasmtime"] }
+sc-network = { version = "0.10.0-dev", git = "https://github.com/subspace/substrate", rev = "1399afdcd8ab4d7fad149c96f9cadcaf04b94f86" }
+sc-service = { version = "0.10.0-dev", git = "https://github.com/subspace/substrate", rev = "1399afdcd8ab4d7fad149c96f9cadcaf04b94f86", features = ["wasmtime"] }
+sc-subspace-chain-specs = { git = "https://github.com/subspace/subspace", rev = "d736cf913f445a8e7dccd5c663e71b19212085ee" }
+serde_json = "1.0.82"
+serde = { version = "1.0.139", features = [ "derive" ] }
+sp-core = { version = "6.0.0", git = "https://github.com/subspace/substrate", rev = "1399afdcd8ab4d7fad149c96f9cadcaf04b94f86" }
+sp-panic-handler = { version = "4.0.0", git = "https://github.com/subspace/substrate", rev = "1399afdcd8ab4d7fad149c96f9cadcaf04b94f86" }
+subspace-core-primitives = { git = "https://github.com/subspace/subspace", rev = "d736cf913f445a8e7dccd5c663e71b19212085ee" }
+subspace-farmer = { git = "https://github.com/subspace/subspace", rev = "d736cf913f445a8e7dccd5c663e71b19212085ee" }
+subspace-fraud-proof = { git = "https://github.com/subspace/subspace", rev = "d736cf913f445a8e7dccd5c663e71b19212085ee" }
+subspace-networking = { git = "https://github.com/subspace/subspace", rev = "d736cf913f445a8e7dccd5c663e71b19212085ee" }
+subspace-rpc-primitives = { git = "https://github.com/subspace/subspace", rev = "d736cf913f445a8e7dccd5c663e71b19212085ee" }
```

Comment:
`subspace-rpc-primitives` seems to be unused too

Reply:
Yes... Removing :)
I ran `cargo udeps`. It was not able to catch it. Are you using any tool for detecting these? I would love to learn it

Reply:
This time I used my :eyes: